SUI/SUI/ContentView.swift

```diff
@@ -4,16 +4,199 @@
+                                HStack {
+                                    Text(Constants.zeroString)
+                                    priceSlider
+                                    Text("\(self.carCost[segmentIndex])")
+                                }
+                                Text("\(Int(firstPrice))")
+                            }
+                        }
+                    }.navigationTitle("\(cars[segmentIndex])")
+                }
+            }
+            HStack {
+                callButton
+                Spacer()
+                chooseButton
+                Spacer()
+                activityButton
+            }
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    private let cars = Constants.carsNames
+    private let typeOfSpoiler = Constants.typeOfSpoilersValues
+    private let customActivity = CustomActivityView(title: Constants.socialString, imageName: Constants.activityImageName) {
+    }
+    @State private var pickerIndex: Float = Constants.zeroFloat
+    @State private var carCost = Constants.carCostsValues
+    @State private var firstPrice: Double = Constants.zeroDouble
+    @State private var offsetX = Constants.zeroInt
+    @State private var isOnEngine = false
+    @State private var isCall = false
+    @State private var isMyChoose = false
+    @State private var segmentIndex = Constants.zeroInt
+    @State private var vEngineIndex = Constants.zeroInt
+    @State private var isEditingSlider = false
+    @State private var isSharePresented = false
+    
+    private var carImage: some View {
+        Image(cars[segmentIndex])
+            .resizable()
+            .frame(width: Constants.imageWidth, height: Constants.imageHeight)
+            .offset(x: CGFloat(offsetX))
+            .cornerRadius(Constants.cornerRadius)
+    }
+    
+    private var carPicker: some View {
+        Picker(selection: Binding(get: {
+            self.segmentIndex
+        }, set: { newValue in
+            self.segmentIndex = newValue
+            self.offsetX = Constants.offsetXNumber
+            self.moveBack()
+        }), label: Text("")) {
+            ForEach(Constants.zeroNumber ..< cars.count) {
+                Text(self.cars[$0]).tag($0)
+            }
         }
+        .pickerStyle(SegmentedPickerStyle())
         .padding()
     }
+    
+    private var spoilerPicker: some View {
+        Picker(selection: $vEngineIndex) {
+            ForEach(Constants.zeroNumber ..< typeOfSpoiler.count) {
+                Text(self.typeOfSpoiler[$0])
+            }
+        } label: {
+            Text(Constants.spolerTitle)
+        }.padding()
+    }
+    
+    private var priceSlider: some View {
+        Slider(value: Binding(get: {
+            firstPrice
+        }, set: { newValue in
+            firstPrice = newValue
+        }), in: 0...Double(carCost[segmentIndex]))
+    }
+    
+    private var callButton: some View {
```

Comment:
```suggestion
    private var callButtonView: some View {
```
ниже везде добавь так же View